content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -44,8 +44,11 @@ Data privacy
 |          | [psycopg2][12]                 | {{< X >}}   |             |
 | **Java**     |                            |             |             |
 |          | jdbc                           | Coming soon | Coming soon |
-| **.NET**     |                            |             |             |
-|          |                                | Coming soon | Coming soon |
+| **.NET** [dd-trace-dotnet][15] >= 2.26.0 ||             |             |
```

Comment:
could you move Java row to the end of the table to keep the `Coming soon` chunk last?

Reply:
Fixed in 69bfe6ece90c7bb51c1f11e37cc6e4d846f98d47